packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -831,3 +847,41 @@ function graphQLErrorWithExtensions(
 ): GraphQLError {
   return new GraphQLError(message, null, null, null, null, null, extensions)
 }
+
+/**
+ * Ideally, all objects should be defined with properties. However, if they are
+ * not, then we can at least stringify the data.
+ *
+ * This function recursively ensures that objects have properties, if not
+ * stringify the data to match the mitigation.
+ */
+function stringifyObjectsWithNoProperties(
```

Comment:
This is the main feature of this PR. Basically, ensure that the properties are defined, otherwise stringify.